cdc-accelerator/constants.js

```diff
@@ -10,6 +10,7 @@ const FEATURE = {
     WEB_SDK: 'webSdk',
     WEB_SCREEN_SETS: 'webScreenSets',
     EMAIL_TEMPLATES: 'emailTemplates',
+    SMS_TEMPLATES: 'smsTemplates',
```

Comment:
This file is from the old architecture and the sms feature is not implemented. This change can be removed